lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb

```diff
@@ -871,7 +871,7 @@ fc_extras
 
     CM_PARSE = re.compile(  r'''
                                 (?P<name>([\w_]+\s*:\s+)+)
-                                (?P<method>[\w_\s]+(?![\w_]*:))\s*
+                                (?P<method>[\w_\s]+(?![\w_]*\s*:))\s*
```

Comment:
Would you mind just checking that the non-greedy form would also work?

```
(?P<name>([\w_]+\s*?:\s+)+)
(?P<method>[\w_\s]+(?![\w_]*\s*?:))\s*?
```

I think it will only make a trivial difference if it does, so I'm not too worried about this.